crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -174,6 +175,64 @@ impl ConfigSettingAccessor<C8yUrlSetting> for TEdgeConfig {
     }
 }
 
+impl ConfigSettingAccessor<C8yHttpSetting> for TEdgeConfig {
+    #[allow(deprecated)]
+    fn query(&self, _setting: C8yHttpSetting) -> ConfigSettingResult<HostPort<443>> {
+        self.data
+            .c8y
+            .http
+            .as_ref()
+            .cloned()
+            .or(self.data.c8y.url.as_ref().cloned().map(ConnectUrl::into))
+            .ok_or(ConfigSettingError::ConfigNotSet {
+                key: C8yUrlSetting::KEY,
+            })
+    }
+
+    fn update(
+        &mut self,
+        _setting: C8yHttpSetting,
+        value: HostPort<443>,
+    ) -> ConfigSettingResult<()> {
+        self.data.c8y.http = Some(value);
+        Ok(())
+    }
+
+    fn unset(&mut self, _setting: C8yHttpSetting) -> ConfigSettingResult<()> {
+        self.data.c8y.http = None;
+        Ok(())
+    }
+}
+
+impl ConfigSettingAccessor<C8yMqttSetting> for TEdgeConfig {
+    #[allow(deprecated)]
+    fn query(&self, _setting: C8yMqttSetting) -> ConfigSettingResult<HostPort<8883>> {
+        self.data
+            .c8y
+            .mqtt
+            .as_ref()
+            .cloned()
+            .or(self.data.c8y.url.as_ref().cloned().map(ConnectUrl::into))
+            .ok_or(ConfigSettingError::ConfigNotSet {
+                key: C8yUrlSetting::KEY,
```

Comment:
Wouldn't be better to tell that `C8yMqttSetting` is not set rather than `C8yUrlSetting`?

```suggestion
                key: C8yMqttSetting::KEY,
```

Reply:
responded in the comment for `C8yHttpSetting`